modules/ossm-member-roll-modify.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll-modify_{context}"]
+= Adding or removing projects from the service mesh
+
+Follow this procedure to modify an existing {ProductShortName} `ServiceMeshMemberRoll` resource using the web console.
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+* The `ServiceMeshMemberRoll` resource is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
+[id="ossm-member-roll-modify-console_{context}"]
+== Modifying the member roll from the web console
+
+.Prerequisites
+* An installed, verified {ProductName} Operator.
+* An existing `ServiceMeshMemberRoll` resource.
+* Name of the project with the `ServiceMeshMemberRoll` resource.
+* Names of the projects you want to add or remove from the mesh.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Installed Operators*.
+
+. Click the *Project* menu and choose the project where your `ServiceMeshControlPlane` is deployed from the list, for example `istio-system`.
+
+. Click the {ProductName} Operator.
```

Comment:
Should all of these `click` items be enclosed in asterisks? Or is there a precedence here?

From: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc

Use single asterisks for web console / GUI items (menus, buttons, page titles, etc.). Use two characters to form the arrow in a series of menu items, ->

Reply:
The formatting is to call out the button name or the menu name.  You don't have to apply formatting to the action (Click, Select).  See the examples down in the Quick Markup Reference section.